docs/prd/GATEWAY-002-test-coverage-improvement-plan.md

```
@@ -0,0 +1,297 @@
+| `authentication.dto.query` | 0% | 0% | 🟡 MEDIUM |
+| `authentication.dto.response` | 0% | n/a | 🟡 MEDIUM |
+| `common.dto.response` | 0% | 0% | 🟡 MEDIUM |
+| `authentication.component` | 99% | 83% | ✅ OK |
+| `authorization.service.command` | 100% | 100% | ✅ OK |
+| `authorization.service.query` | 100% | 100% | ✅ OK |
+| `authorization.dto.command` | 100% | 100% | ✅ OK |
+| `authorization.dto.response` | 100% | n/a | ✅ OK |
+
+#### 작업 목록
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)